floss/main.py

```diff
@@ -543,6 +542,60 @@ def main(argv=None) -> int:
 
     language = identify_language(sample=sample, static_strings=static_strings)
 
+    if language == Language.GO:
+        enhancedStaticStrings = extract_go_strings(sample=sample, min_length=args.min_length)
+        enhancedStaticStrings = list(enhancedStaticStrings)
+
+        # Calculate String Coverage
+
+        all_static_string_list = []
+
+        for string_obj in static_strings:
+            if string_obj.string.isprintable():
```

Comment:
which non-printable sequences are included here? we should cover what the static string algorithms extracts, so may have to change the check in the go extraction code?